samples/assistant/csharp-ooproc/AssistantApis.cs

```diff
@@ -56,33 +57,14 @@ public class CreateChatBotOutput
     /// HTTP POST function that sends user prompts to the assistant chat bot.
     /// </summary>
     [Function(nameof(PostUserQuery))]
-    public static async Task<PostResponseOutput> PostUserQuery(
+    public static async Task<HttpResponseData> PostUserQuery(
         [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "assistants/{assistantId}")] HttpRequestData req,
-        string assistantId)
+        string assistantId,
+        [AssistantPostInput("{assistantId}", "{Query.message}", Model = "%CHAT_MODEL_DEPLOYMENT_NAME%")] AssistantState state)
     {
-        string? userMessage = await req.ReadAsStringAsync();
-        if (string.IsNullOrEmpty(userMessage))
-        {
-            HttpResponseData badResponse = req.CreateResponse(HttpStatusCode.BadRequest);
-            await badResponse.WriteStringAsync("Request body is empty");
-            return new PostResponseOutput { HttpResponse = badResponse };
-        }
-
-        HttpResponseData response = req.CreateResponse(HttpStatusCode.Accepted);
-
-        return new PostResponseOutput
-        {
-            HttpResponse = response,
-            ChatBotPostRequest = new AssistantPostRequest { UserMessage = userMessage, Id = assistantId }
-        };
-    }
-
-    public class PostResponseOutput
-    {
-        [AssistantPostOutput("{assistantId}", Model = "%CHAT_MODEL_DEPLOYMENT_NAME%")]
-        public AssistantPostRequest? ChatBotPostRequest { get; set; }
-
-        public HttpResponseData? HttpResponse { get; set; }
+        HttpResponseData response = req.CreateResponse(HttpStatusCode.OK);
+        await response.WriteAsJsonAsync(state.RecentMessages.FirstOrDefault()?.Content);
```

Comment:
Should this be `LastOrDefault()`, just in case we change our mind about whether to return intermediate messages in the future?